app/forms/support_interface/conditions_form.rb

```diff
@@ -0,0 +1,85 @@
+module SupportInterface
+  class ConditionsForm
+    class OfferConditionField
+      include ActiveModel::Model
+
+      attr_accessor :id, :text
+
+      validates :text, length: { maximum: MakeAnOffer::MAX_CONDITION_LENGTH, too_long: ->(c, _) { "Condition #{c.id + 1} must be %{count} characters or fewer" } }
```

Comment:
If the [OfferValidations](https://github.com/DFE-Digital/apply-for-teacher-training/blob/master/app/services/offer_validations.rb#L4) service can't be used to check the conditions length, can we reference the `MAX_CONDITIONS_LENGTH` constant from there, as we will be deprecating MakeAnOffer soon.

Reply:
Yes, the extended limit is on the further conditions, as some vendor integrations can't separate conditions so they send them as a single condition, thus the leniency with condition 1